### DISCUSSION TITLE
Fehler cannot debit

### DISCUSSION THREAD

**Discussion by Tanja8611** [iotaledger/legacy-migration-tool#140] [Help & Support]
Ich kriege immer diesen Fehler angezeigt wenn ich mein Iota Guthaben von alten Wallet zu den neuen Überweise. Kann mir dabei jemand helfen.

**Comment by theregenfarmer** [comment]
I have the same problem....

**Comment by vbgweb** [comment]
Hi, same problem here. Hope for some news and solution. Thanks!

**Comment by Sarah-** [comment]
Same problem...
errorMessage: cannot debit (base tokens: 1071598042) from xxxx@iotaxxxx: not enough funds

**Comment by ap-a11y** [comment]
Same problem here...

**Just found this on the topic:**
_Note, if you have lots of small amounts of $IOTA (<1Mi) spread across lots of addresses, it may not be possible to migrate these small amounts, particularly if their total is less than the minimum required storage deposit on the Stardust network._

Found here: https://blog.iota.org/iota-legacy-migration-tool/

Amount of IOTA is displayed correctly, but the value is displayed as <$0.01 ("Confirm the new address") and $0.00 ("Migrate your funds")

Log can be provided if helpful

![Migration_error](https://github.com/user-attachments/assets/3caa5c03-f93d-4f01-96c3-7eda9810dae8)

**Comment by JuanJoseNacher** [comment]
Yo tengo el mismo problema  me muestra la cantidad de iotas y cuando intento hacer la migración me pone un error de fondos insuficientes.

**Comment by Benchwalk** [comment]
I have the same problem:
migration error: cannot debit (base tokens: 499500000) from ----@iota---- not enough funds

![2024-12-03 22_06_13-IOTA Legacy Migration Tool](https://github.com/user-attachments/assets/bd7f699d-a4da-4362-b735-be35a8aebd98)

**Comment by vbgweb** [comment]
Same here!!

**Comment by oxamberge1** [comment]
Hat sich denn noch keine Lösung gefunden?

**Comment by amin3z777** [comment]
I have same problem

**Comment by theregenfarmer** [comment]
Due to discord the problem is known since 2 weeks. No solution yet...

**Comment by begonaalvarezd** [comment]
The issue `errorMessage: cannot debit (base tokens: XXX) from YYY: not enough funds` has been mitigated, please try the migration again. We apologize for the troubles :pray:

**Comment by lusybasti** [comment]
Thank you @begonaalvarezd for your efforts. I had the same error and tried again. My balance shows now 0 in the migrataion tool . Even multiple "checks again" didn't help. The balance remains 0. I hope you can help me/us.

**Reply by begonaalvarezd** [reply]
There is an issue in the tool fetching the fiat price, if your find IOTAs but it says that they are valued 0$ you can ignore that, the important part is the IOTAs found. 
If you still cant find your IOTAs, you should either keep searching or try to trace your address in case you already migrated. 
I suggest you join our discord if you need further support https://discord.iota.org/

**Reply by lusybasti** [reply]
Thank you for your quick response. I mean actually the IOTA balance and not the display of the $. The IOTA balance was displayed correctly before the bug. I'll keep trying and if necessary look for help in the discord.

**Reply by lusybasti** [reply]
Thanks a lot. I had a mistake in the passphrase.

**Comment by sirecki** [comment]
Unfortunately, the error has not been fixed. I try to migrate with current software and actually get the same error. How can I reopen the case?

**Reply by begonaalvarezd** [reply]
The migrator has enough funds to proceed with the migrations, which error are you getting exactly?

**Reply by sirecki** [reply]
I have opened an issue for that: https://github.com/iotaledger/legacy-migration-tool/issues/147
errorMessage: cannot debit (base tokens: 797701000) from 05204969@iota1pphx6hnmxqdqd2u4m59e7nvmcyulm3lfm58yex5gmud9qlt3v9crs9sah6m

**Comment by mwendig** [comment]
the problem still exists .... I have exactly the same problem, can somebody help / support?

**Comment by wbulot** [comment]
Can confirm the issue is still there.

**Comment by glagolef** [comment]
I have the same problem. Should this issue be reopened or do we need to create a new one?

|depositAddress: {
  "trytes": "TRANSFERKXNECDW9YWSENCXBMDSCSEHVYWSZCYWXDDTXIXPYFWQBTARWWZQZD9C9NWWWU9REJAPWBCOA9JNZCT9ERD",
  "bech32": "iota1qza85mlujfl5gvmef9lcrykaek4hp29ch7dzc9ytu8dsgqu8jra8ut559h6"
} 
balance: 1004000000 
errorMessage: cannot debit (base tokens: 1004000000) from 05204969@iota1pphx6hnmxqdqd2u4m59e7nvmcyulm3lfm58yex5gmud9qlt3v9crs9sah6m: not enough funds

**Comment by vittod** [comment]
I can confirm the same problem is yet unsolved...

balance: 506991000 
errorMessage: cannot debit (base tokens: 506991000) from ...: not enough funds

**Comment by Andy0709wejobbr** [comment]
i have still the same issue... can you please fix it?

balance: 419000000 
errorMessage: cannot debit (base tokens: 419000000) from XXXXXXXXXXXXXXX: not enough funds

**Comment by kirankn** [comment]
Still seeing the issue. Is there a fix being planned?

**Reply by Andy0709wejobbr** [reply]
the bug was reopened and they are working on it. please comment in this new thread:
https://github.com/iotaledger/legacy-migration-tool/discussions/149
